### PR TITLE
depart du token

### PR DESCRIPTION
**Pull request by pablo9797**
token got in the front, hook useState used in app.js to to use token state in all redirection

### CODE REVIEW

File: back/server.js [hunk 1]
```diff
@@ -92,3 +92,56 @@ app.listen(port, err => {
...
+				token
+			})
+		})
+	}
+	else {
+		res.status(400).send();
+	}
+})
+
+function verifyToken(req, res, next){
```
Comment (ghost) [inline back/server.js:131]: You already created a verifyToken function earlier (line 20), this one seems to be complete so you might want to delete the previous one

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by pablo9797** [comment]
conflict resolved

File: back/server.js (hunk 1 above)
Comment (pablo9797) [reply back/server.js:131]: it s ok now

Comment (ghost) [reply back/server.js:131]: Did you push it ?

**Comment by pablo9797** [comment]
ok push from token

**Pull request closed without merging**